
# Diagnostic Analysis and Reporting Tool 

## Finding the one in a million 
Rare und undiagnosed diseases are often caused by genetic defects. Rapid identification of the causal mutation(s) provides a molecular diagnosis that can inform potentially invasive treatment decisions and reduces the need for lengthy (and often costly) testing. Genetic diagnoses are currently hampered by the large number of potential disease genes, by the high number of variants identified and by the complexity of variant annotation, which can lead to major delays in reaching a confident genetic diagnosis.

## Reporting variants
DART is a user-friendly, non-commercial software tool that assists researchers and clinicians with the filtering and interpretation of genetic variants obtained by next generation sequencing (e.g. diagnostic panels, exome sequencing, or whole genome sequencing. DART facilitates confident genetic diagnoses in compliance with emerging community standards. Starting with an annotated VCF file, DART supports interactive filtering, implements ACMG- AMP classification, provides full audit history, and helps generate custom patient reports.

# Going the extra mile 
Starting with an annotated VCF file, DART supports the user by providing

- Sample focused analysis 
- Virtual gene panels
- Standardized filtering
- Interactive exploration and visualization of variants
- Disease focused variant analysis
- Trio / relatedness variant representation
- Support ACMG-AMP variant curation

# Full control 
DART rivals commercial variant annotation software in terms of features and user-friendliness, and – as an open source software with no strings attached – it puts the full control over genetic data back in the hands of the clinical laboratories that analyse the data. It thus avoids undue commercialization of genome data and facilitate responsible sharing. 

## Access
A public test version is available on https://www.biomedical-sequencing.at/pub-dart/login 
For installation / configuration details, please see GITHUB.


<!-- # Welcome to MkDocs

For full documentation visit [mkdocs.org](https://www.mkdocs.org).

## Commands

* `mkdocs new [dir-name]` - Create a new project.
* `mkdocs serve` - Start the live-reloading docs server.
* `mkdocs build` - Build the documentation site.
* `mkdocs -h` - Print help message and exit.

## Project layout

    mkdocs.yml    # The configuration file.
    docs/
        index.md  # The documentation homepage.
        ...       # Other markdown pages, images and other files. -->
